code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -105,11 +105,13 @@
 	item_flags = NOSLIP
 	slowdown = SHOES_SLOWDOWN+1
 	species_restricted = null
+	style = STYLE_NEG_HIGH
```

Comment:
Bruh.

Reply:
noslip swimming flippers do not appear to be particularly stylish to me, and I think of the walk they would give you on land as unstylish.